src/wallet_new.ts

```diff
@@ -4,12 +4,15 @@ import * as crypto from 'crypto';
 import jwkToPem, { JWK } from 'jwk-to-pem';
 import Arweave from 'arweave';
 import * as mnemonicKeys from 'arweave-mnemonic-keys';
+import { Bytes } from './ardrive';
```

Comment:
Note for the future: we should push some of these typealiases down into a separate file to avoid circular dependencies among the source files.